### DISCUSSION TITLE
ESP8266 multiple DS18b20 (onewire/1wire) on different KNX group addresses

### DISCUSSION THREAD

**Discussion by samiwe** [arendst/Tasmota#17720] [Q&A]
hi at all, 
I am running an ESP8266 (12.3.1 KNX) with 2 DS18b20 onewire sensors. I want to send the temperature to KNX, but to different group addresses. But I can only select 1x temperature and link it to a group address.

As a test I have attached the 2nd ds18b20 to its own GPIO, but even then I get no possibility to link it to its own group address.

In the webinterface both temperatures (no matter if connected in series to one GPIO or each to an own GPIO) are displayed correctly.

Can someone tell me if multiple temperatures to multiple group addresses is possible at all ?

If yes, how should I proceed?

Thanks & greetings
sam

**Comment by arendst** [comment]
Currently ons supports but one temperature sensor.

**Comment by samiwe** [comment]
Hi Theo,
thanks for your reply and a super big thanks for this awesome piece of software called tasmota.

that means at the moment only 1x temperature to 1x group address is supported !?

Will this be upgraded to multiple group addresses in the (near) future ?

sam

**Reply by arendst** [reply]
Alas not in the forseeable future.
